include/RAJA/policy/cuda/reduce.hpp

```diff
@@ -408,6 +408,9 @@ RAJA_DEVICE RAJA_INLINE T block_reduce(T val, T identity)
   // reduce per warp values
   if (numThreads > policy::cuda::WARP_SIZE) {
 
+    static_assert(policy::cuda::MAX_WARPS <= policy::cuda::WARP_SIZE,
```

Comment:
Just to double check, do we have a way to test this?

Reply:
I'm not sure what you mean exactly, this is a static assert so it will fail at compile time if WARP_SIZE and MAX_WARPS change to invalid values.

Reply:
Yes, I was wondering under what conditions we would trigger the assert. Doing a quick grep it looks like the values are hard-coded, so to trigger the assert we would have to manually change `MAX_BLOCK_SIZE` and `WARP_SIZE` . I was thinking the values would get picked up when we configure RAJA or something like that.

Reply:
Yep, if we ever change them we'll know to come back and fix this.